pages/docs/reference/js-ir-compiler.md

```diff
@@ -0,0 +1,65 @@
+---
+type: doc
+layout: reference
+category: "JavaScript"
+title: "Using the experimental IR compiler"
+---
+# Using the experimental Kotlin/JS IR compiler
+
+The (currently still experimental) Kotlin/JS IR compiler backend is the main focus of innovation around Kotlin/JS paves the way forward for the technology. Its goals are to improve on pain points that the previous compiler had, in areas of developer experience, size of generated code (through dead code elimination), and to improve interoperability with the JavaScript and TypeScript ecosystems, to name just a few.
+
+The experimental IR compiler backend is available starting with Kotlin 1.4 through the Kotlin/JS Gradle plugin. To enable it in your project, pass a compiler type to the `js` function in your Gradle build script:
+
+<!--suppress ALL -->
+<div class="sample" markdown="1" mode="groovy" theme="idea">
+
+```groovy
+kotlin {
+    js(IR) { // or: LEGACY, BOTH
+        // . . .
+    }
+    binaries.executable()
+}
+```
+
+</div>
+
+- `IR` uses the new IR compiler backend for Kotlin/JS.
+- `LEGACY` uses the default compiler backend.
+- `BOTH` compiles your project with the new IR compiler as well as the default compiler backend. This is mainly useful for authoring libraries that are compatible with both backends, see [below](#authoring-libraries-for-the-ir-compiler-with-backwards-compatibility).
+
+The compiler type can also be set in the `gradle.properties` file, with the key `kotlin.js.compiler=ir`. (This behaviour is overwritten by any settings in the `build.gradle(.kts)`, however).
+
+## (Current) limitations of the IR compiler
+
+A major change with the new IR compiler backend is the absence of binary compatibility with the default backend. A lack of such compatibility between the two backends for Kotlin/JS means that a library created with the new IR compiler backend can’t be used from the default backend, and vice versa.
```

Comment:
Can you somehow highlight key phrases for each limitation so that people see them when quickly scanning the page.
For example: 
- A major change with the new IR compiler backend is **the absence of binary compatibility** with the default backend.
- Currently, the IR backend **does not generate source maps** for Kotlin code.
And similar for other limitations.